CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.6.8
+
+* Require the latest version of `package:http_parser`.
```

Comment:
If users just run "pub get", they'll also get the older `shelf` version, so I don't think there's much help there.